examples/Compass/CompassAzimuth/CompassAzimuth.ino

```diff
@@ -0,0 +1,34 @@
+// Библиотека для работы с модулями IMU
+#include <TroykaIMU.h>
+
+// Создаём объект для работы с иагнитометром/компасом
+Compass compass;
+
+// Калибровочные значения, полученные в калибровочной матрице
+// из примера compassCalibration
+const float compassCalibrationBias[3] = { 567.893, -825.35, 1061.436 };
+
+const float compassCalibrationMatrix[3][3] = { { 1.909, 0.082, 0.004 },
```

Comment:
In the CompassCalibration example, I get three raw values from the `readX()`, `readY()`, `readZ()` methods. It is not clear which of this refers to what? What to write to `compassCalibrationBias` and what to `compassCalibrationMatrix`? This comment, on the other hand, misleads the user. Because it has to use those raw values as guided by our wiki article. Better then leave a link to the article in the comments.